cli/README.md

```diff
@@ -1,55 +1,74 @@
+
+#### Windows
+
+Paste the following command into your terminal and hit enter.
 
-Almost all commands can work in two different modes: globally or scoped to a project. You can configure a project with `xata init`. This will create a `.xatarc` file (although `.xatarc.json` and `package.json` are also supported) and an `.env` file.
+```powershell
+iwr https://xata.io/install.ps1 -useb | iex
+```
 
-- The project configuration file contains some preferences and information, including the `databaseURL`.
-- The `.env` file contains an API key and optionally a fallback branch name (more about the fallback branch in the [Git integration](#git-integration) section).
+## Authentication
 
-When working in _global mode_, commands won't know which database you want to use, because there's no project configuration file, and thus, no `databaseURL` configured. So, you can either pass a `--db` flag to specify the database URL, or just don't pass any argument and you will be prompted interactively to select a workspace, then a database, and depending on the command, a branch name.
+To authenticate your CLI installation, run `xata auth login`. Here, you will be presented with a couple of options:
 
-In order to configure authentication for the _global mode_, you have to authenticate first using `xata auth login`. This will guide you through the process to generate a new global API key that will be stored in `~/.config/xata/credentials`.
+- **Create a new API Key** will open your browser and, when you're logged in to Xata, allow you to create a new [API key](https://docs.xata.io/concepts/api-keys) for use with the CLI.
+- **Use an existing API Key** will prompt for an existing key you have, which you can paste into your terminal. We recommend isolating keys per use-case, so creating a new one might be a good idea here.
 
-## Configuring a project
+## Initializing a Project
 
-In ordert to configure the CLI in a project you'll run `xata init --db=https://{workspace}.xata.sh/db/{database}`. The flag `--db` is not mandatory but recommended, because this way you won't be asked for the workspace/database and thus, a global API key is not required.
+To initialize a new database, run `xata init` in your project's directory.
 
-This command will ask you a few questions and will run a few operations:
+- if you don't have a [workspace](https://docs.xata.io/concepts/workspaces), you will be prompted to create one.
+- if you have workspaces, you'll be asked to choose one, or to create a new one.
 
-- It will create a project configuration file (`.xatarc`).
-- It will create or update your `.env` file to store a database-scoped API key. (At the time of writing these docs it'll actually store a personal API key, but this will change soon).
-- It will ask you if you want to install the SDK and/or use the TypeScript/JavaScript code generator.
```

Comment:
I think this information was very relevant.

Reply:
Done in https://github.com/xataio/client-ts/pull/400/commits/b463f7b00fb60d1da95d400bb457ce1749ba5534